examples/morpheusvm/tests/integration/transfer_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package integration_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/vm"
+	"github.com/ava-labs/hypersdk/tests/integration"
+	"github.com/ava-labs/hypersdk/tests/workload"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+var _ = integration.RegisterTest("Transfer Transaction", func(t ginkgo.FullGinkgoTInterface, tn workload.TestNetwork) error {
```

Comment:
So the only difference between integration tests and e2e tests are how the transactions are confirmed under the hood. This just adds unnecessary work for the VM developer, and I'd like to remove this code duplication since the testing logic is exactly the same.

Reply:
Can we create the test registry in one place and support passing the registry into both the integration and e2e environment?

This way VM test writers, can write and register their tests in a single place and then pass the registry in via both the integration and e2e entrypoints to run the VM specific tests in either environment.